core/tests/puppeteer-acceptance-tests/blogpostAdmin-tests/blog-post-publish-unpublish.js

```diff
@@ -0,0 +1,21 @@
+const e2eBlogPostAdmin = require("../utility-functions/blogPostAdminUtils.js");
+const testConstants = require("../utility-functions/testConstants.js");
+
+const homePage = testConstants.Dashboard.MainDashboard;
+const blogDashboardUrl = testConstants.URLs.BlogDashboard;
+
+async function publishBlogAsBlogPostAdmin() {
+  const blogPostAdmin = await new e2eBlogPostAdmin();
+  await blogPostAdmin.openBrowser();
+
+  await blogPostAdmin.signInWithEmail("testadmin@example.com");
+  await blogPostAdmin.waitForPageToLoad(homePage);
+  await blogPostAdmin.goto(blogDashboardUrl);
+  await blogPostAdmin.createNewBlogPostByTitle("Test Blog Post");
+  await blogPostAdmin.publishNewBlogPost();
```

Comment:
Just to check, are they doing all these actions from the blog dashboard (not the blog editor)?